src/main/java/com/exactpro/th2/crawler/SessionAliasType.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2021 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.th2.crawler;
+
+public enum SessionAliasType {
+    PLAIN_TEXT("PLAIN_TEXT"),
+    REGEXP("REGEXP");
+
+    private final String stringValue;
+
+    SessionAliasType(String stringValue) {
+        if (!stringValue.equals("PLAIN_TEXT") && !stringValue.equals("REGEXP")) {
+            throw new IllegalArgumentException("SessionAliasType parameter must be either \"PLAIN_TEXT\" or \"REGEXP\"");
+        }
```

Comment:
Why do you need this validation? Nobody can create a new enum value outside this file